js/apps/admin-ui/src/clients/authorization/policy/Group.tsx

```diff
@@ -164,7 +149,6 @@ export const Group = () => {
             </Tbody>
           </Table>
         )}
-        {errors.groups && <FormErrorText message={t("requiredGroups")} />}
```

Comment:
This seems to be the only place the `requiredGroups` label is being used, so it can be removed from the properties files.